dracut/modules.d/99asahi-firmware/module-setup.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/sh
+# SPDX-License-Identifier: MIT
+
+# called by dracut
+check() {
+    if [ -n "$hostonly" ] && [ ! -e /proc/device-tree/chosen/asahi,efi-system-partition ]; then
+       return 0
+    elif [ -z "$hostonly" ]; then
+        return 0
+    else
+       return 255
+    fi
+}
+
+# called by dracut
+depends() {
+    echo fs-lib
+    return 0
+}
+
+# called by dracut
+installkernel() {
+    instmods apple-mailbox nvme-apple
+}
+
+# called by dracut
+install() {
+    asahiscriptsdir="/usr/share/asahi-scripts"
+    inst_dir $asahiscriptsdir
+    $DRACUT_CP -R -L -t "${initdir}/${asahiscriptsdir}" "${dracutsysrootdir}${asahiscriptsdir}"/*
+    inst_multiple cut dirname modprobe seq sha1sum sleep tar
```

Comment:
I think you can drop some of these things now (at least sha1sum and cut, maybe dirname too? and tar)

Reply:
`sha1sum` and `tar` are indeed not used anymore, so I dropped them. I did, however, add `cpio`, `mount`, and `umount`.